sage_codes/Safe Curve code

```diff
@@ -0,0 +1,385 @@
+/// This section shows all of the proofs from their 
```

Comment:
Comments should be represented with `#` since this is not a .rs file the `///` has not much sense.